src/app/pages/ArticlePage/PagePromoSections/utility/index.js

```diff
@@ -0,0 +1,14 @@
+export default ({ sectionType, assetUri, uri, contentType, index = 0 }) => {
```

Comment:
I've no idea what this function does as it has no name and is inside of a folder called `utility`

I'd give it a name and then also give the directory the same name

Reply:
Renamed as below. Same facepalm moment as earlier :D